charts/wasp-routing-service-v2/Chart.yaml

```diff
@@ -0,0 +1,27 @@
+annotations:
+  category: wasp
+  licenses: Apache-2.0
+apiVersion: v2
+appVersion: 2.1.2
+dependencies:
+  - condition: kafka.enabled
+    name: kafka
+    repository: https://charts.bitnami.com/bitnami
+    version: 20.x.x
+  - name: common
+    repository: https://charts.bitnami.com/bitnami
+    tags:
+      - bitnami-common
+    version: 2.x.x
```

Comment:
I am unsure about whether or not we should fork this chart.

Reply:
I wouldn't. They seem to version sensibly and this chart is very stable